### ISSUE TITLE
Translation from POV to GSV Image Coordinates

### ISSUE THREAD

**Issue by shokiami** [ThatOneGoat/sidewalk-cv-2021#51]
Due to faulty translation from POV to GSV Image coordinates, some label positions are slightly off (both images are of "curb ramp" labels):
![52235_0](https://user-images.githubusercontent.com/43970567/147007242-de5b1e1c-50ea-42ef-bd06-2a8e1762592e.jpg)
![912_0](https://user-images.githubusercontent.com/43970567/147007287-30e67427-1fae-4078-90f1-7868544f62f0.jpg)

More information about the issue and our proposed fix can be found here:
https://docs.google.com/document/d/1TLCz07VBthmeUFLU6HcYn6Zt4DIzHgzX6jquemegECs/edit?usp=sharing

**Comment by jonfroehlich** [comment]
Thanks for making this an issue @shokiami. Any progress on this? It will be a huge win once fixed!

**Comment by shokiami** [comment]
Yes. Although, we are currently stuck. As Michael mentioned on the doc, our proposed fix assumes that we are only looking at a portion of the pano. However, since we are looking at the entire panorama, our FOV is 360 degrees, which leads to an infinite focal length...
We are figuring it out though! I found a post that seems promising: https://stackoverflow.com/questions/43741885/how-to-convert-spherical-coordinates-to-equirectangular-projection-coordinates
